Readme.md

```diff
@@ -80,9 +80,11 @@ Example for using custom scripts stored in the `scripts` directory:
 PATH=$PATH:./scripts fwanalyzer -cfg system_fwa.toml -in system.img -out system_check_output.json
 ```
 
-The [devices](devices/) folder contains helper scripts for unpacking and dealing with specific devices types and firmware package formats such as [Android](devices/android).
+The [devices/](devices/) folder contains helper scripts for unpacking and dealing with specific devices types and firmware package formats such as [Android](devices/android).
 It also includes general configuration files that can be included in target specific FwAnalyzer configurations.
 
+Check.py in the [devices/](devices) folder provides a universal script to effectively use FwAnalyzer, see [devices/Readme.md](devices/Readme.md) for details. For most people that is what you will use FwAnalyzer with.
```

Comment:
Can we be more descriptive of how it's used? Maybe "This is how most people will invoke FwAnalyzer" or something.

At first glance I don't know what check.py is for.